.github/workflows/ci.yml

```diff
@@ -16,7 +16,20 @@ jobs:
       run: pip install -r tools/requirements.txt
     - name: check .rst document syntax
       run: tools/check-rst-syntax.sh
-    - name: generate and check intrinsic specs (rst)
+    - name: setup nvm
+      shell: bash -l -eo pipefail {0}
+      run: nvm install 8.10.0 && nvm use 8.10.0
+    - name: add config package dependencies
+      uses: actions/cache@v2
+      with:
+        path: ~/.npm
+        key: npm-${{ hashFiles('package-lock.json') }}
+        restore-keys: npm-
+    - name: install ci dependencies
+      run: npm ci --ignore-scripts
+    - name: install markdown-link-check
+      run: npm install -g markdown-link-check
```

Comment:
What I meant was to move these steps inside the script ` tools/generate-intrinsics-specs.sh`, right before invoking `markdown-link-check`